docs/blog/oras-0.14-and-future.md

```diff
@@ -0,0 +1,73 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_Feynman Zhou, CNCF Ambassador, Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the CLI tool on Aug 22. ORAS v0.14 provides four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provide new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing. 
+
+This blog will demonstrate how to use ORAS v0.14 to copy an image from a validated artifact registry to personal conintainer registry, then attach a SBOM to it and discover the graph.
+
+
+## Install ORAS 0.14
+
+Install the latest release of ORAS on a Linux machine:
+
+```
+curl -LO https://github.com/oras-project/oras/releases/download/v0.14.1/oras_0.14.1_linux_amd64.tar.gz
+mkdir -p oras-install/
+tar -zxf oras_0.14.1_*.tar.gz -C oras-install/
+mv oras-install/oras /usr/local/bin/
+rm -rf oras_0.14.1_*.tar.gz oras-install/
+```
+
+## Copy an image from Registry A to Registry B
+
+In this demo, assume all images in are validated in MAR, so I will use ORAS to copy the container image from Microsoft Artifact Registry (MAR) to my personal repository of Docker Hub. You can use your prefered container registry with ORAS.
+
+```
+./oras copy mcr.microsoft.com/mmlspark/spark2.4:1.0.0 registry-1.docker.io/pengfeizhou/spark2.4:1.0.0
+```
+
+## Using SBOM Tool to generate a SBOM
+
+[SBOM Tool](https://github.com/microsoft/sbom-tool) is a general purpose, enterprise-proven, build-time SBOM generator open-sourced by Microsoft.
+
+Install SBOM Tool on a Linux machine:
+
+```
+curl -Lo sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
+chmod +x sbom-tool
+```
+
+Generate a SBOM for the Spark image stored in Docker Hub:
+
+```
+./sbom-tool generate -di registry-1.docker.io/pengfeizhou/spark2.4:1.0.0 -b ./foo -pn bar -pv 0.1 -bc ./foo -ps MyCompany -nsb http://mycompany.com
+```
+
+## Attach the SBOM to this image
+
+Attach the generate SBOM to this Spark image stored in Docker Hub:
+
+```
+oras attach registry-1.docker.io/pengfeizhou/spark2.4:1.0.0 foo/_manifest/spdx_2.2/manifest.spdx.json --artifact-type example/sbom
```

Comment:
love the `attach` command
Same comment about removing registry-1 from the docker.io reference.